app/src/main/res/menu/mediaplayer.xml

```diff
@@ -32,6 +32,7 @@
         android:id="@+id/audio_controls"
         android:icon="@drawable/ic_sliders"
         android:title="@string/audio_controls"
+        android:visible="false"
         custom:showAsAction="always">
```

Comment:
I would add `custom:showAsAction="never"` here and remove the icon (both from the xml file here and from the Java code that makes the menu item visible). The item does not need to be displayed so prominently anymore. That then also makes room for other icons. I think this is the only use of the icon, so it might be possible that the CI then complains that the icon file is unused. In that case, you can safely delete the icon file :)

Reply:
Done. And it did complain so I've removed the icon file as well.